docs/cli-reference/dfx-cycles.md

```diff
@@ -90,6 +90,7 @@ You can specify the following options for the `dfx cycles transfer` command.
 
 | Option                           | Description                                                        |
 |----------------------------------|--------------------------------------------------------------------|
+| `--top-up <principal>`           | The canister to which you want to transfer cycles.                 |
```

Comment:
```suggestion
| `--top-up <principal>`           | The canister which you want to top up.                 |
```
I think it's a bit hard to figure out what the difference between `--to-owner` and `--top-up` is if you don't know the terminology will. May be worth expanding a bit in the description above too

Reply:
I agree, though at this point I'm not sure we're doing ourselves a favor by combining these two operations into a single command.